bin/pulumilocal

```diff
@@ -124,22 +193,79 @@ def set_localstack_pulumi_config(args: argparse.Namespace):
 def main():
     # Parse arguments from call to pulumi CLI that set the stack name and directory
     parser = argparse.ArgumentParser(add_help=False)
-    parser.add_argument("command", help="pulumi primary command",
+    parser.add_argument("command", help="Pulumi command",
                         type=str)
-    parser.add_argument("-s", "--stack", help="pulumi stack name",
+    parser.add_argument("-s", "--stack", help="The name of the stack to operate on. Defaults to the current stack",
                         required=False,
                         type=str)
-    parser.add_argument("-C", "--cwd", help="run in this directory",
+    parser.add_argument("-C", "--cwd", help="Run pulumi as if it had been started in another directory",
                         required=False,
                         type=str)
+    parser.add_argument("--config-file", help="Use the configuration values in the specified file rather than detecting the file name",
+                        required=False,
```

Comment:
Thanks for adding clarity to these help strings, these are much better.

Reply:
tbh I didn't feel the need to do all nice help messages as there is no way you'd see any of these, but thanks for beautify them :)